spec/features/admin_adds_video_spec.rb

```diff
@@ -0,0 +1,48 @@
+require 'spec_helper'
+
+feature "Admin adds video" do
+  given(:admin) { Fabricate(:admin) }
+  given(:user) { Fabricate(:user) }
+
+  background do
+    Fabricate(:category, title: 'comedy')
+    Fabricate(:category, title: 'action')
+  end
+
+  scenario "admin signs in and add video with correct input" do
+    sign_in(admin)
+
+    visit new_admin_video_path
+    fill_in "Title", with: 'Up'
```

Comment:
you could extract this logic (the page interactions) into methods to make your spec read better and all be at the same level of abstraction.